packages/dropdown-react/src/Select.tsx

```diff
@@ -45,14 +47,18 @@ export function Select({
     };
     const componentClassName = "jkl-dropdown".concat(
         inline ? ` jkl-dropdown--inline` : "",
+        forceCompact ? ` jkl-dropdown--compact` : "",
         !!errorLabel ? ` jkl-dropdown--invalid` : "",
         value === "" ? ` jkl-dropdown--no-value` : "",
         className ? ` ${className}` : "",
     );
-    const labelClassName = variant ? ` jkl-label--${variant}` : "";
+    const labelClassName = "jkl-label".concat(
+        variant ? ` jkl-label--${variant}` : "",
+        forceCompact ? ` jkl-label--compact` : "",
+    );
     return (
-        <label className={componentClassName}>
-            <span className={"jkl-label" + labelClassName}>{label}</span>
+        <label data-testid="jkl-dropdown" className={componentClassName}>
+            <span className={labelClassName}>{label}</span>
```

Comment:
This stuff gets repeated in a lot of components now. It feels like we need a separate`Label` component, maybe?

Reply:
This gets fixed in #442!